translations/stripes-acq-components/en.json

```diff
@@ -20,7 +20,13 @@
   "fundDistribution.code": "Code",
   "fundDistribution.encumbrance": "Encumbrance",
   "fundDistribution.id": "Fund ID",
-  "fundDistribution.percent": "Percent",
+  "fundDistribution.value": "Value",
+  "fundDistribution.type": "Select adjustment",
+  "fundDistribution.type.sign.percent": "%",
+  "fundDistribution.type.sign.amount": "$",
```

Comment:
Do you really need to translate the signs `$` and `%`?

Reply:
it's like labels, I suppose they could be different per country/language